Location.h

```diff
@@ -1,13 +1,27 @@
 #pragma once
 
 #include <string>
+#include "LocationId.h"
 
 class Location
 {
 public:
-   Location(std::string description);
+   Location(std::string description,
+            int north,
+            int east,
+            int south,
+            int west);
+
    std::string get_description();
+   int get_north();
+   int get_east();
+   int get_south();
+   int get_west();
 
 private:
    std::string m_description;
```

Comment:
should all of this shit be const?

Reply:
const location? no way! I will want it to evolve, have objects and things added / removed

Reply:
oh you mean get_north etc... ye probably - when i crack that const nut!